Translation note for Home/end to avoid mistranslations.

NekoJonez opened the pull request:
## Description
Should fix #35608 , this is my first time I did a translation note ... So feedback is welcome. 

## Types of changes
Line 134 added.

NekoJonez committed:
Translation note for Home/end to avoid mistranslations.

Should fix #35608 , this is my first time I did a translation note ... So feedback is welcome. Line 134 added.

Mamaduka commented on packages/components/src/date-time/index.js:
```diff
@@ -131,6 +131,7 @@ function DateTimePicker(
 								<abbr aria-label={ __( 'Home and End' ) }>
 									{ __( 'Home/End' ) }
 								</abbr>
+								/* Translators: Home/End reffer to the 'Home' and 'End' buttons on the keyboard.*/
```
```suggestion
								<abbr aria-label={ __( 'Home and End' ) }>
									{ /* Translators: Home/End reffer to the 'Home' and 'End' buttons on the keyboard.*/ }
									{ __( 'Home/End' ) }
								</abbr>
```

NekoJonez replied:
@Mamaduka Implemented. :)

NekoJonez committed:
Merge branch 'WordPress:trunk' into patch-1

NekoJonez committed:
Translation note placement improved and some enters removed

Thx to the feedback, I placed the translation note on a better place. 
I also formatted the code a bit better so that it looks like the last one.

Mamaduka commented:
Hi, @NekoJonez

Try running `npm run format-js` after the changes, it should fix all the linting issues.

NekoJonez commented:
> Hi, @NekoJonez
> 
> Try running `npm run format-js` after the changes, it should fix all the linting issues.

I only work in the browser... So, running commands isn't possible for me :/

Mamaduka committed:
Fix lint errors

Mamaduka reviewed (approved):
No worries, I fixed those on my end 👍 

I will try to merge this once all checks are green. Thanks for contributing, and congratulations on your first PR 🎉

NekoJonez commented:
> No worries, I fixed those on my end 👍
> 
> I will try to merge this once all checks are green. Thanks for contributing, and congratulations on your first PR 🎉

Excellent! Thanks my man! 
And happy to help. 

Btw, is this translation note also going to WP core? Since, the strings are already in core and it would be a shame if we avoid mistranslations in Gutenberg and not in Core for the same strings and features.

Mamaduka commented:
> Btw, is this translation note also going to WP core?

Yes, this change will be part of 5.9.

NekoJonez commented:
> > Btw, is this translation note also going to WP core?
> 
> Yes, this change will be part of 5.9.

Lovely! A polygot job well done me thinks!
